src/libs/fileDownload/FileUtils.js

```diff
@@ -159,9 +159,12 @@ function appendTimeToFileName(fileName) {
  *
  * @param {String} path - the blob url of the locally uplodaded file
  * @param {String} fileName
+ * @param {Function} onSuccess
+ * @param {Function} onFailure
+ *
  * @returns {Promise}
  */
-const readFileAsync = (path, fileName) =>
+const readFileAsync = (path, fileName, onSuccess, onFailure) =>
```

Comment:
@DylanDylann What about making the `onFailure` optional with default value of `() => {}`? For now in 2 out of 3 usages we basically doing it, making it optional would benefit on readability in various usage scenarios